modules/apps/frontend-js/frontend-js-web/test/liferay/util/portlet_url/create_action_url.ts

```diff
@@ -12,17 +12,15 @@
  * details.
  */
 
-'use strict';
-
-import createActionURL from '../../../../src/main/resources/META-INF/resources/liferay/util/portlet_url/create_action_url.es';
+import createActionURL from '../../../../src/main/resources/META-INF/resources/liferay/util/portlet_url/create_action_url';
 
 describe('Liferay.Util.PortletURL.createActionURL', () => {
 	it('returns a URL object with a href parameter containing the p_p_lifecycle parameter set to 1', () => {
 		Liferay = {
 			ThemeDisplay: {
 				getPortalURL: jest.fn(() => 'http://localhost:8080'),
 			},
-		};
+		} as any;
```

Comment:
Mocking can get tricky... 🤔

Reply:
Yes. But that's expected. That's why you have to use libraries like mockito in Java, too :shrug:

Reply:
Maybe using `as const` can be more secure, at least we can have the viability of what we have inside the Liferay mock.